public/index.html

```diff
@@ -152,6 +152,258 @@ <h1>Nurture.</h1>
+          A mobile operating system based on a modified version of the Linux kernel and other open source 
+          software, designed primarily for touchscreen mobile devices such as smartphones and tablets.
+        </p>
+      </div>
+      <!-- Temporary pic -->
+      <img src="./assets/ipad.png" class="product-pic" >
+      <!--  -->
+    </div>
+
+    <div class="gadgets-2">
+      <div class="gadgets-2-header">
+        <h1>In everyone lies an innovator.</h1>
+        <p>
+          Here at Developer Student Clubs Loyola, we believe anyone can be a
+          change-maker.
+        </p>
+      </div>
+      <div class="gadgets-2-body">
+        <img src="./assets/ipad.png" class="laptop-2">
+        <img src="./assets/mobile.png" class="mobile-2">
+        <img src="./assets/ipad.png" class="ipad-2">
+      </div>
+    </div>
+
+    <div class="recweek-app">
+      <img src="./assets/recweek-app.png" class="recweek-app-pic">
+      <div class="recweek-app-body">
+        <p>PROJECTS</p>
+        <h1>Pavilion</h1>
+        <p>Created by the Developer Student Clubs Executive Board</p>
+        <div class="recweek-app-tech">
+          <img src="./assets/firebase.png" class="firebase-icon">
+          <img src="./assets/flutter.png" class="flutter-icon">
+        </div>
+        <p>
+          Developer Student Clubs (DSC) Loyola is a student-led tech organization inside the 
+          Ateneo De Manila University. The Katipunan chapter is one among the many Google 
+          Developer-powered communities that is sparking massive change beyond borders, 
+          cross-country. DSC's devotion to technological literacy explains its
+        </p>
+        <div class="recweek-app-links">
+          <a href="#"><img src="./assets/google.png" class="google-btn"></a>
+          <a href="#"><img src="./assets/apple.png" class="apple-btn"></a>
+        </div>
+      </div>
+    </div>
+
+    <div class="qwiklabs">
+      <div class="qwiklabs-body">
+        <p>PROJECTS</p>
+        <h1>Qwiklabs Finishers PH</h1>
+        <p>Created by Franz Taborlupa, Tanzell Go, Rafael Dytoc, Lenz Dagohoy, Jared de Guzman, and Shaina Loira</p>
+        <div class="qwiklabs-tech">
+          <img src="./assets/firebase.png" class="firebase-icon">
+        </div>
+        <p>
+          Developer Student Clubs (DSC) Loyola is a student-led tech organization inside the 
+          Ateneo De Manila University. The Katipunan chapter is one among the many Google 
+          Developer-powered communities that is sparking massive change beyond borders, 
+          cross-country. DSC's devotion to technological literacy explains its
+        </p>
+        <div class="qwiklabs-links">
+          <a href="#" class="visit-btn">View Site</a>
+        </div>
+      </div>
+      <img src="./assets/ipad.png" class="qwiklabs-pic">
+    </div>
+
+    <div class="bantay-bayan">
+      <!-- Temporary pic -->
+      <img src="./assets/ipad.png" class="bantay-bayan-pic">
```

Comment:
**add alt text**

Alt text uses:
1. Adding alternative text to photos is first and foremost a principle of web accessibility. Visually impaired users using screen readers will be read an alt attribute to better understand an on-page image.

2. Alt tags will be displayed in place of an image if an image file cannot be loaded.

3. Alt tags provide better image context/descriptions to search engine crawlers, helping them to index an image properly.

Checkout https://moz.com/learn/seo/alt-text#:~:text=Adding%20alternative%20text%20to%20photos,image%20file%20cannot%20be%20loaded.